test/unit/validators/dependency-audit.test.js

```diff
@@ -744,6 +744,120 @@ describe('dependency-audit Unit Tests', () => {
     });
   });
 
+  describe('doVersContainGitRepository method', () => {
```

Comment:
Question about `dependency-audit.test.js`: why we need this tests since it’s duplicate tests from `test/unit/rules/*`?

Now I carefully write code and tests at the same style as already written. But for me it’s a little bit overtested :)

Reply:
Hey @miripiruni - great question. You are right that it is probably a bit overkill. I figure it is better to be safe 🤓 . I'll think about adjustments for v5.